js3.js

```diff
@@ -143,4 +171,10 @@ a(() => {
 //and returns the result. Then, create a test in js3.test.js that tests it and use
 //the anonymous function to invoke it.
 
-module.exports = { myPerson, objectExtend, Student }
\ No newline at end of file
+function functionInAFunction(otherFunction){
+    return otherFunction()
+}
+
+// Not certain what an anonymous function is, I hope this is correct.
```

Comment:
My mistake for not highlighting that clearly enough. A function without a name is an anonymous function.
`function() {}` or `() => {}` are anonymous since they aren't declared with a name.